### PR TITLE
Made metrics port configurable in operator

### PR DESCRIPTION
**Pull request by jroper**

### CODE REVIEW

File: Dockerfile.js-shopping-cart [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:8.15
+FROM node:12
```
Comment (jroper) [inline Dockerfile.js-shopping-cart:1]: While this change has nothing to do with this PR, node 12 is needed by the js shopping cart `package.json`, and I did need to rebuild shopping cart to test the knative support which was what this change was done for.

File: cloudstate-operator/manifests/cloudstate-operator.yaml [hunk 2]
```diff
@@ -1930,6 +1930,8 @@ data:
       image: cloudstateio/cloudstate-proxy-postgres:latest
     cassandra:
       image: cloudstateio/cloudstate-proxy-cassandra:latest
+    sidecarMetrics:
+      port: 9092
```
Comment (pvlugter) [inline cloudstate-operator/manifests/cloudstate-operator.yaml:1934]: 9099?

**Review by pvlugter — APPROVED** [review]
LGTM

File: cloudstate-operator/go.mod [hunk 3]
```diff
@@ -20,7 +20,7 @@ require (
 	go.uber.org/atomic v1.5.1 // indirect
 	go.uber.org/multierr v1.4.0 // indirect
 	go.uber.org/zap v1.13.0 // indirect
-	golang.org/x/net v0.0.0-20200520004742-59133d7f0dd7
+	golang.org/x/net v0.0.0-20200520004742-59133d7f0dd7 // indirect
```
Comment (pvlugter) [inline cloudstate-operator/go.mod:23]: Build failing on git diff check on this line:

https://app.circleci.com/pipelines/github/cloudstateio/cloudstate/564/workflows/a1f27613-1b50-40da-9834-1d01b10cf8e5/jobs/1767

Comment (jroper) [reply cloudstate-operator/go.mod]: Yeah I'm not sure what triggers that change to happen, when I run exactly the same commands that the build runs, it doesn't remove this line. I just removed it manually.

Comment (pvlugter) [reply cloudstate-operator/go.mod]: Same for me. Didn't remove the line. Different go versions?

Comment (jroper) [reply cloudstate-operator/go.mod]: Not sure, changing it manually fixed it.

**Pull request merged**